requirements.txt

```diff
@@ -1,278 +1,209 @@
-toml==0.10.1 \
-    --hash=sha256:bda89d5935c2eac546d648028b9901107a595863cb36bae0c73ac804a9b4ce88 \
-    --hash=sha256:926b612be1e5ce0634a2ca03470f95169cf16f939018233a670519cb4ac58b0f
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --generate-hashes requirements.in
```

Comment:
the `requirements.in` argument is unnecessary, as it's automatically detected. it's still preserved when passed, so removing it here means you don't have to type it when you update it.
```suggestion
#    pip-compile --generate-hashes
```

Some dependencies have python-version-specific requirements, and if a dev runs a globally installed `pip-compile`, it may not be on the same version of python as the docker container. So I recommend enforcing the correctness of this in CI, e.g.
https://github.com/mozilla/bigquery-etl/blob/3725a64/.circleci/config.yml#L37-L46

Reply:
I had seen this when I was looking at how you used pip-tools, but then completely forgot to add it. Thank you! This will save hours of time at some point in the future when I'm all confused about what's going on.